ide/che-core-ide-app/src/main/java/org/eclipse/che/ide/connection/WsConnectionListener.java

```diff
@@ -44,6 +44,10 @@ public WsConnectionListener(EventBus eventBus,
             @Override
             public void onWorkspaceStarted(WorkspaceStartedEvent workspace) {
                 messageBus = messageBusProvider.getMessageBus();
+                if (messageBus == null) {
+                    Log.error(WsConnectionListener.class, "Unable to add onClose handler. Message bus must not ber null.");
```

Comment:
But, you're committing into IDE 5. And this commit will be present in this version. Fix a typo.